v6.0 Desktop Docs (#8474)
mattermost/docs: source/product-overview/desktop-app-changelog.md

```{include} common-esr-support.md
```

(release-v6-0)=
## Release v6.0

**Release Day:** November 14, 2025

**Download Binaries:** [Mattermost Desktop on GitHub](https://github.com/mattermost/desktop/releases/latest)

```{Note}
Mattermost Desktop App v6.0.0 contains low severity level security fixes. Upgrading is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
```

### Compatibility

- Desktop App is supported on any currently supported [Mattermost server version](https://docs.mattermost.com/product-overview/mattermost-desktop-releases.html#latest-releases).
- Updated Chromium minimum supported version to 140+ and updated Windows minimum supported version to v11+.

### Improvements

#### Windows

- Switched to ``registry-js`` native module for Windows registry reading.

#### All Platforms

> - Added the ability to open multiple tabs and windows for any given server.

Reviewer: For an image, the one on this PR should be fine: https://github.com/mattermost/desktop/pull/3491

As for verbiage, no opinion. We could have AI sum up the points for us.

- Added parent-child popout window support, with listeners to facilitate communication.

> - Added pre-auth headers to the server modal and an inject header into all requests to that server.

Reviewer 2: ```suggestion
- Added pre-auth headers to the server modal, this header is injected into all requests to that server.
```

Mattermost Desktop App v6.0.0 contains low severity level security fixes. Upgrading is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
```

### Compatibility

- Desktop App is supported on any currently supported [Mattermost server version](https://docs.mattermost.com/product-overview/mattermost-desktop-releases.html#latest-releases).
- Updated Chromium minimum supported version to 140+ and updated Windows minimum supported version to v11+.

### Highlights

#### Desktop App Multi-View Architecture
 - Mattermost Desktop App v6.0 introduces a new Multi-View Architecture that replaces the previous fixed-tab layout. This enhancement introduces a dynamic, flexible system where users can open, close, and manage multiple views or windows, including any part of the Mattermost app (e.g., Channels, Boards, Playbooks). This architectural update provides a more customizable and efficient workspace experience while laying the foundation for advanced multi-window and pop-out features in future releases.

> .. image:: ../images/desktop-multi.png

Reviewer 3: I don't think this is valid markdown.
